logbook-core/src/main/java/org/zalando/logbook/KeyValueHttpLogFormatter.java

```diff
@@ -0,0 +1,15 @@
+package org.zalando.logbook;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class KeyValueHttpLogFormatter extends AbstractPreparedHttpLogFormatter {
+
+    @Override
+    public String format(final Map<String, Object> content) {
+        return content.entrySet().stream()
+                .map(entry -> String.format("%s=%s", entry.getKey(), entry.getValue()))
```

Comment:
Any escaping that needs to be done for values that contain a `=` e.g. the URL?

Reply:
Well, that is a idiomatic question.
Splunk for example is eating such format like `uri=http://host?k=v` and allows you to search it. But also during the syntax highlighting, it suggest two pairs `uri=http://host` and `k=v`, which allow you to search those parameters bound to each other (full search) or independently (partial search). It is quite useful, when you have many parameters in uri, but you is interested in only some.